website_docs/getting_started/browser.md

```diff
@@ -199,3 +213,27 @@ Now, rebuild your application and open the browser again. In the console of the
   ]
 }
 ```
+
+### Add Plugins
+
+If you want to instrument AJAX requests, User Interactions and others, you can add plugins to your
+instrumentation:
+
+```javascript
+registerInstrumentations({
+  instrumentations: [
+    new UserInteractionInstrumentation(),
+    new XMLHttpRequestInstrumentation()
+  ],
+});
+```
+
+## Meta Packages for Web
+
+To leverage the most common instrumentations all in one you can simply use the
+[OpenTelemetry Meta Packages for Web](https://www.npmjs.com/package/@opentelemetry/auto-instrumentations-web)
+
+## Instrumentation with Browser Extension
+
+If you'd like to get a quick preview on how your website (or any other site) would look like with OpenTelemetry
```

Comment:
Maybe a bit confusing wording to some people as the instrumentation doesn't change how the website looks(well unless it breaks it :P). Maybe mention that this is about previewing what data is collected.

Reply:
```suggestion
If you'd like to quickly preview what instrumentation would look like with your website (or any other site)
```

Reply:
Yeah, I was not able to phrase this properly, thanks for the suggestion!